content/blog/2022/11/ui-1.0.0-release.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Kubewarden UI 1.0.0 is released!
+authors:
+- Jordon Leach 
+date: 2022-11-01 # TODO: Update date before publishing
+---
+
+We are excited to announce that the Kubewarden UI 1.0.0 has been released!
```

Comment:
I would like to point users to the artifacts (Helm charts, or repo tag), and a link to the docs on how to install the Kubewarden UI. This way they can read the blog and land on their feet with the UI, without investigating.

Reply:
I added the release and documentation for the installation (this link will not work until https://github.com/kubewarden/docs/pull/147 is merged). I created actual releases which links to the helm chart, these can be found from the gh-pages link `https://kubewarden.github.io/ui`.